### PR TITLE
RHINENG-12214 Update CPU and Memory Param names

### PR DESCRIPTION
**Pull request by saltgen**
We need to update CPU and Memory params in the Clowdapp file.

This is to ensure we utilize resources on ephemeral environments optimally.

## Why do we need this change? :thought_balloon:

Please include the __context of this change__ here. If the change depends on Kruize add details about that as well!

## Documentation update? :memo:

- [ ] Yes
- [x] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Does this change depend on specific version of Kruize
  - If yes what is the version no:
  - [ ] Is that image available in production or needs deployment?
- [ ] Bugfix
- [ ] New Feature
- [x] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Please check JIRA for additional details.

### CODE REVIEW

**Comment by saltgen** [comment]
Test commit removed
image

**Comment by saltgen** [comment]
Jenkins success run, https://ci.int.devshift.net/job/RedHatInsights-ros-backend-pr-check/951/console

File: clowdapp.yaml [hunk 1]
```diff
@@ -28,11 +28,11 @@ objects:
         args: ["-c", "export FLASK_APP=manage.py && poetry run flask db upgrade && poetry run flask seed && poetry run gunicorn ros.api.main:app"]
         resources:
           requests:
-            cpu: ${CPU_REQUEST}
-            memory: ${MEMORY_REQUEST}
+            cpu: ${CPU_REQUEST_ROSRHEL}
```
Comment (r14chandra) [inline clowdapp.yaml:31]: Are you going to create an MR on the app-interface side? You would need to update it in the [deploy.yml](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/insights/ros/deploy-clowder.yml) as well.

**Review by r14chandra — COMMENTED** [review]
You would need to update the same in the [deploy.yml](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/insights/ros/deploy-clowder.yml) file in app-interface as well.

File: clowdapp.yaml (hunk 1 above)
Comment (saltgen) [reply clowdapp.yaml:31]: Yes, those PRs are ready, I will share the same with you as soon as I merge these app side ones

**Review by r14chandra — APPROVED** [review]

**Pull request merged**